docs/usage.md

```diff
@@ -1,12 +1,25 @@
 # Usage
 
-## 1 - Online RL Training
-Horizon supports online training environments for model testing. To train a model on OpenAI Gym, simply run:
+Horizon is designed for large-scale, distributed recommendation/optimization tasks where we don't have access to a simulator.  In this environment, it's typically better to train offline on batches of data, and release new policies slowly over time.  Because the policy updates slowly and in batches, we use *off-policy* algorithms.  To test a new policy without deploying it, we rely on *counter-factual policy evaluation (CPE)*, a set of techniques for estimating a policy based on the actions of another policy.
```

Comment:
Nice addition. I wonder if you could add newlines to make it text-editor friendly (keep it <100 chars)?